tap_mysql/__init__.py

```diff
@@ -404,12 +405,17 @@ def row_to_singer_record(stream, version, row, columns):
             boolean_representation = elem != b'\x00'
             row_to_persist += (boolean_representation,)
 
+        elif 'boolean' in column_type or column_type == 'boolean':
```

Comment:
Again, should we do case-insensitive comparison? I'm not sure if we're doing case sensitive or insensitive comparison in the rest of the tap but either way it would be a shame if some MySQL server reported types in another case and that broke things.

Reply:
@mdelaurentis maybe i shouldn't have named the var `column_type` -- here we're looking at the JSON Schema property type (which we set lowercase here https://github.com/singer-io/tap-mysql/pull/38/files#diff-1463e5d5f4c92b29cdaace9a7519ea79R206)